output-validation-rule.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+var assert = require('assert');
+var Joi = require('joi');
+
+module.exports = OutputValidationRule;
+
+function OutputValidationRule(status, spec) {
+  assert(status, 'OutputValidationRule: missing status param');
+  assert(spec, 'OutputValidationRule: missing spec param');
+
+  this.ranges = status.split(',').map(trim).filter(Boolean).map(rangify);
+  this.status = status;
+  this.spec = spec;
+
+  this.validateInput();
+}
+
+/**
+ * Validates it's input values
+ *
+ * @throws Error
+ */
+
+OutputValidationRule.prototype.validateInput = function validateInput() {
+  assert(/^[0-9,\-]+|\*$/.test(this.status),
```

Comment:
Also to note, the original regex would allow things like `0,` or even `,` as well as `1048,4282,205818333`. Not to mention what @jlai said where you could put any arbitrary values after one valid `0-9,\-` character.